Platform/AlderlakeBoardPkg/Library/FspmUpdUpdateLib/FspmUpdUpdateLib.c

```diff
@@ -606,6 +609,7 @@ UpdateFspConfig (
         Fspmcfg->FirstDimmBitMask = 0x0;
         Fspmcfg->SkipCpuReplacementCheck = 0x0;
         break;
+#endif
```

Comment:
why building ADL_PS out when enabling ADL_N?

Reply:
because ADL-N boardid is conflict with ADL-PS so putting this conditional check to choose either one of them to avoid the conflict.